### PR TITLE
doc/foundation: add ceph foundation info here

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: doc/foundation.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+-------
+
+* `China Mobile <https://www.chinamobileltd.com/>`_
+* `DigitalOcean <https://www.digitalocean.com/>`_
+* `Intel <http://www.intel.com/>`_
+* `OVH <https://www.ovh.com/>`_
+* `Red Hat <https://www.redhat.com/>`_
+* `SoftIron <https://www.softiron.com/>`_
+* `SUSE <https://www.suse.com/>`_
+* `Canonical <https://www.canonical.com/>`_
```
Comment (jdurgin) [inline doc/foundation.rst]: if this is alphabetical it's in the wrong spot

**Comment by tchaikov** [comment]
might want to drop "python-common: move pytest integration from setup.py to tox.ini " from this PR.

**Comment by liewegas** [comment]
jenkins render docs

**Comment by liewegas** [comment]
jenkins render docs

**Comment by liewegas** [comment]
retest this please

**Pull request merged**